### DISCUSSION TITLE
Podcast Feature Request - Description Required

### DISCUSSION THREAD

**Discussion by carolineschettler** [forem/forem#16905] [Feature Requests]
**Issue:**
When a user goes to submit a podcast, the way the form currently works, they can fully submit without adding a description of their podcast. However, to publish on our end, we're unable to publish a podcast without a description, leaving the podcast unpublished, floating around in our system.

**Feature:**
It would be great to require users to write a description of their podcast before they can submit it for our approval to publish. 

**Example:** 
[This podcast ](https://dev.to/admin/content_manager/podcasts/738/edit) was submitted without a description - checking the publish button and hitting Update Podcast reroutes to the empty description box. Ideally, they shouldn't have been able to submit it without filling out the description field.

**Comment by michael-tharrington** [comment]
Great suggestion, Caroline! 

You're absolutely right that there's a disconnect here. If we require podcasts to have a description in order for admin to publish them, it totally makes sense that we would require submitters to include the description, otherwise it's on the admin to make one up or contact the submitter and ask.
